packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -237,13 +227,6 @@ contract SuperchainConfig is Initializable, ISemver {
         emit ConfigUpdate(UpdateType.REMOVE_SEQUENCER, abi.encode(_sequencer));
     }
 
-    /// @notice Sets the system owner address.
-    /// @param _systemOwner The new system owner address.
-    function _setSystemOwner(address _systemOwner) internal {
-        Storage.setAddress(SYSTEM_OWNER_SLOT, _systemOwner);
-        emit ConfigUpdate(UpdateType.SYSTEM_OWNER, abi.encode(_systemOwner));
```

Comment:
Interesting, now we lose this event, i think that is fine

Reply:
As is the only place it was emitted was in the initializer.